packages/sources/sportsdataio/src/sport/mlb/endpoint/schedule.ts

```diff
@@ -0,0 +1,43 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { ExecuteWithConfig } from '@chainlink/types'
+import { Config } from '../../../config'
+import { ethers } from 'ethers'
+import { GameResponse } from '../types'
+import { getGamesByDate } from '../utils'
+
+export const NAME = 'schedule'
+
+const customParams = {
+  date: true,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, customParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const date = validator.validated.data.date
+
+  const games = await getGamesByDate(date, config)
+  const encodedGames = encodeGames(games)
+  const respData = {
+    data: {
+      encodedGames,
+      games,
+      result: encodedGames.length,
+    },
+    result: encodedGames.length,
```

Comment:
Why are we setting `result` to `encodedGames.length`?
Would it not be better to have a different key like `count` to make this more explicit?

Reply:
We could.  I didn't know what else to set `result` to.  Any suggestions?

Reply:
Ah, in that case I'd prefer just `games` or `encodedGames`

Reply:
Can `result` be of any type or must it be a primitive value?

Reply:
It should be possible for it to be of any type, it just limits what the CL node can do with it without transforming it in any way. Considering this will be used by TOML job specs, I assume, then that shouldn't be a problem.

Reply:
alright I'll make this change